src/ec/suite_b/ecdsa.rs

```diff
@@ -50,6 +66,12 @@ enum ECDSAVerificationAlgorithmID {
     ECDSA_P384_SHA384_FIXED,
 }
 
+impl PartialEq for ECDSAVerificationAlgorithm {
+    fn eq(&self, other: &Self) -> bool { self.id == other.id }
+}
```

Comment:
Well, there is `PartialEq` implementation for `ECDSASigningAlgorithm`, so implementing it for `ECDSAVerificationAlgorithm` only benefits the symmetricity. Also, `signature::VerificationAlgorithm` is a very elegant single-method protocol, which I'm a big fan of, and which modern cryptographic libraries lack, so I think coercing it to struct is a bad idea, but it may just be my irrational thinking:)